src/Components/ADT3DViewer/ADT3DViewer.stories.tsx

```diff
@@ -22,7 +23,10 @@ export default {
 
 const mockSceneId = 'f7053e7537048e03be4d1e6f8f93aa8a';
 
-export const Engine = (_args, { globals: { theme, locale } }) => {
+export const Engine: ComponentStoryFn<typeof ADT3DViewer> = (
```

Comment:
can we change it to the new style of decorating for theme and localization and using ComponentStory instead of ComponentStoryFn? not sure the difference though...
<img width="380" alt="image" src="https://user-images.githubusercontent.com/45217314/211061398-9a46467c-2689-4829-99b6-458230cb6480.png">

Reply:
Looks like ComponentStoryFn === ComponentStory, in our Storybook version
![image](https://user-images.githubusercontent.com/55852250/211085044-12f535a3-7d04-4b97-a419-7686d1c692cd.png)

Reply:
ohh interesting